conference/cfp/models.py

```diff
@@ -31,21 +31,39 @@ class Submission(models.Model):
     updated_at = models.DateTimeField(
         _('Updated'), auto_now=True
     )
+    notes = models.TextField(_('Notes'))
+    selected = models.BooleanField(_('Selected'), default=False)
     proposal_title = models.CharField(
         _('Title'), max_length=400
     )
     proposal_abstract = models.TextField(
         _('Abstract')
     )
+    proposal_audience = models.CharField(
+        _('Audience'), choices=AUDIENCE, max_length=10,
+    )
+
+    class Meta:
+        verbose_name = _('talk submission')
+        verbose_name_plural = _('talk submissions')
+
+    def __str__(self):
+        return '{} "{}" by {}'.format(self._meta.verbose_name, self.proposal_title, self.author)
+
+    class Meta:
+        abstract = True
+
+    def __str__(self):
+        return '{} "{}" by {}'.format(self._meta.verbose_name, self.proposal_title, self.author)
+
+
+class TalkSubmission(AbstractSubmission):
```

Comment:
@EmadMokhtar it's better to keep the model name unchanged, to keep the existing data, or generate the migrations to only rename the model instead of dropping the old one and create the new one